src/slack/sendDispatch.js

```diff
@@ -22,6 +30,14 @@ const sendPrimaryRequestInfo = async (record, text, reminder) => {
   return res;
 };
 
+/**
+ * Send secondary request info.
+ *
+ * @param {object} record The request record object.
+ * @param {string} text The message to send to slack.
+ * @param {object} threadTs The message threads object returned from slack.
```

Comment:
`threadTs` is a string returned from the slack API:
https://api.slack.com/methods/chat.postMessage